keps/sig-api-machinery/5073-declarative-validation-with-validation-gen/README.md

```diff
@@ -682,241 +735,20 @@ A number of the rules in the below sections are not implemented but will be triv
-   <td style="background-color: null">
-    feature gate is disabled
-   </td>
-   <td style="background-color: null">
-    `+k8s:ifOptionDisabled(FeatureX)=&lt;if-disabled-validator-tag>`
-   </td>
-   <td style="background-color: null">
-    N/A
-   </td>
-  </tr>
-  <tr>
-   <td style="background-color: null">
-    validate each key
-   </td>
-   <td style="background-color: null">
-    `+k8s:eachKey=&lt;eachKey-validator-tag>`
-   </td>
-   <td style="background-color: null">
-    N/A
-   </td>
-  </tr>
-  <tr>
-   <td style="background-color: null">
-    validate each value
-   </td>
-   <td style="background-color: null">
-    `+k8s:eachVal=&lt;eachVal-validator-tag>`
-   </td>
-   <td style="background-color: null">
-    N/A
-   </td>
-  </tr>
-  <tr>
-   <td style="background-color: null">
-    uniqueness
-   </td>
-   <td style="background-color: null">
-    `+k8s:listType=&lt;type>`
-   </td>
-   <td style="background-color: null">
-    `x-kubernetes-list-type`
-   </td>
-  </tr>
-  <tr>
-   <td style="background-color: null">
-    shared struct fields (subfield)
-   </td>
-   <td style="background-color: null">
-    `+k8s:subfield(subField-json-name)=&lt;subfield-validator-tag>`
-   </td>
-   <td style="background-color: null">
-    N/A
-   </td>
-  </tr>
-  <tr>
-   <td>
-    union member (Discriminated/Non-Discriminated)
-   </td>
-   <td>
-    +k8s:unionMember={"union": ""}
-   </td>
-   <td>
-    x-kubernetes-unions
-   </td>
-  </tr>
-  <tr>
-   <td>
-    union discriminator
-   </td>
-   <td>
-    +k8s:unionDiscriminator={"union": ""}
-   </td>
-   <td>
-    x-kubernetes-unions
-   </td>
-  </tr>
-  <tr>
-   <td style="background-color: null">
-    immutable after set
-   </td>
-   <td style="background-color: null">
-    `+k8s:immutable`
-   </td>
-   <td style="background-color: null">
-    N/A
-   </td>
-  </tr>
-  <tr>
-   <td style="background-color: null">
-    required once set
-   </td>
-   <td style="background-color: null">
-    `+k8s:requiredOnceSet`
-   </td>
-   <td style="background-color: null">
-    N/A
-   </td>
-  </tr>
-  <tr>
-   <td style="background-color: null">
-    immutable(frozen) at creation
-   </td>
-   <td style="background-color: null">
-    `+k8s:frozen`
-   </td>
-   <td style="background-color: null">
-    N/A
-   </td>
-  </tr>
-  <tr>
-   <td style="background-color: null">
-    group membership (virtual field)
-   </td>
-   <td style="background-color: null">
-    `+k8s:memberOf(group: &lt;groupname>)`
-   </td>
-   <td style="background-color: null">
-    N/A
-   </td>
-  </tr>
-  <tr>
-   <td style="background-color: null">
-    list map item reference (virtual field)
-   </td>
-   <td style="background-color: null">
-    `+k8s:listMapItem(list-map-key-field-name: value,...])`
-   </td>
-   <td style="background-color: null">
-    N/A
-   </td>
-  </tr>
-</table>
+
+| **Type of validation** | **IDL tag** | **Relative OpenAPI validation field** | **Stability Level (in 1.35 release) ** |
```

Comment:
Currently this table drops a number of the entries that existed in the original table. I believe the table currently has some out of date entries but perhaps dropping entries can be done a separate PR.  If you want to change this table from an html table to a markdown table to add the stability levels that is ok but I don't think we should drop any of the existing entries.